src/components/Tasks/Sidebar.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+
+import { useSelector, useDispatch } from 'react-redux';
+
+import List from '@material-ui/core/List';
```

Comment:
We have our own wrapper for List in `widgets`. If it's not too much work, it would be great if you could change to use that for more consistency in the code (we use it everywhere else too). No worries if it's a lot of hassle.

Reply:
That's fine, I can change it to that.

I was actually curious, why are we using this custom List component instead of the built in Material UI one?

Reply:
As you can see by the code you had to write to use it there are a lot of components involved when creating list items, which in turn meant abysmal performance when using it in large lists. This was even worse with earlier versions of material-ui.
In order to solve it we just implemented our own list widget that looked the same and was orders of magnitude faster.

As of a few days ago (5cb8dda9cce94abe4aa57735548960706bce61c1) we changed back to use the material-ui widgets internally because we finally got around to adding virtualized lists, which fixes the performance problem altogether.

Reply:
Ahh that makes sense. That mui library is quite bulky haha

Reply:
Is it ok if I change the widget? There is no support for `selected` yet

Reply:
Sure thing.